nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmUploadHandlerSupport.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.helm;
+
+import java.util.Set;
+
+import org.sonatype.nexus.repository.rest.UploadDefinitionExtension;
+import org.sonatype.nexus.repository.security.ContentPermissionChecker;
+import org.sonatype.nexus.repository.security.VariableResolverAdapter;
+import org.sonatype.nexus.repository.upload.UploadDefinition;
+import org.sonatype.nexus.repository.upload.UploadHandlerSupport;
+import org.sonatype.repository.helm.internal.util.HelmAttributeParser;
+
+import static org.sonatype.repository.helm.internal.HelmFormat.NAME;
+
+/**
+ * @since 1.0.11
```

Comment:
Is this the discussed release version or it should be `1.next `here?

Reply:
Same for some other files in this PR

Reply:
I have not seen this strategy used for community tho I've seen this asked a lot=)
I think there's no automation for community so it's a manual set.  Would be interested to learn otherwise and if it's project specific.